src/Microsoft.AspNetCore.Razor.Runtime/project.json

```
@@ -4,7 +4,9 @@
   "compilationOptions": {
     "allowUnsafe": true,
     "keyFile": "../../tools/Key.snk",
-    "warningsAsErrors": true
+    "warningsAsErrors": true,
+    "nowarn": [ "CS1591" ],
```

Yes. To suppress errors for missing XML docs for which we haven't added.